ci/release/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# shellcheck shell=bash
+
+set -euo pipefail
+
+npx --yes \
```

Comment:
what are the rules for how a release version is decided?

Reply:
https://github.com/semantic-release/semantic-release#how-does-it-work

Reply:
cool, thanks. So I guess we have to be careful to add the `BREAKING CHANGE:` footer to a commit?

I see that `buf` has a `breaking` command, perhaps we can run that as part of CI and fail if the correct footer isn't present?

Reply:
That's a great idea!